packages/theme-patternfly-org/pages/global-css-variables.md

```diff
@@ -42,6 +42,12 @@ Where...
 - `breakpoint` is a media query breakpoint such as `sm` for `$pf-global--breakpoint--xs`.
 - `pseudo-element` is either `before` or `after`.
 
+### Breakpoint variables and class suffixes
+
+PatternFly defines a number of standard breakpoints. These are always used as a min-width breakpoint; i.e. using the `-md` breakpoint would apply to everything at the `-md` width and wider.
```

Comment:
Dunno if it's worth noting but the table is the exception to this.

```suggestion
PatternFly defines a number of standard breakpoints. These are always used as a `min-width` breakpoint; i.e. using the `-md` breakpoint would apply to everything at the `-md` width and wider.
```

Reply:
@srambach @mcoker the table is the exception bc we're changing the default presentational behavior and breaking the built in semantics by making it `display: grid`. It doesn't make sense to assign `display: table` to a `table`, so this is an appropriate application of `min-width` media query as the table resumes its normal behavior at whatever breakpoint.